test/ci/utils.sh

```diff
@@ -480,12 +481,12 @@ set_or_update_owner_redirect_info() {
   local real_owner_ip
   real_owner_ip="$(get_real_ip "${owner_service_name}")"
   log_info "Checking if 'RVTO2Addr' is configured on owner side (${owner_url})"
-  if [ -z "$(get_owner_redirect_info "${owner_url}")" ]; then
+  if [ -z "$(get_ownerinfo "${owner_url}")" ]; then
     log_warn "'RVTO2Addr' not found, creating it"
-    set_owner_redirect_info "${owner_url}" "${real_owner_ip}" "${owner_dns}" "${owner_port}" "${owner_protocol}"
+    set_ownerinfo "${owner_url}" "${real_owner_ip}" "${owner_dns}" "${owner_port}" "${owner_protocol}"
   else
     log_info "'RVTO2Addr' found, updating it"
-    update_owner_redirect_info "${owner_url}" "${real_owner_ip}" "${owner_dns}" "${owner_port}" "${owner_protocol}"
+    update_ownerinfo "${owner_url}" "${real_owner_ip}" "${owner_dns}" "${owner_port}" "${owner_protocol}"
```

Comment:
It's OK if you rename the functions but do the same in `scripts/fdo-utils.sh`: you added new functions not renamed and modified the existing ones.